test/Microsoft.NET.Restore.Tests/GivenThatWeWantToUseFrameworkRoslyn.cs

```diff
@@ -24,17 +24,15 @@ public void It_restores_Microsoft_Net_Compilers_Toolset_Framework_when_requested
             var testAsset = _testAssetsManager
```

Comment:
Can't comment on it but then we should change the name of this test now to `It_downloads...`